bazel/linux/qemu.bzl

```diff
@@ -127,3 +132,102 @@ See the RuntimeBundleInfo provider for details.
         }
     ),
 )
+
+def _kernel_qemu_aarch64_run(ctx):
```

Comment:
making a copy of this rule does not feel right.  perhaps this can be parameterized and a common rule impl can be used for x86 and aarch64 ?

Reply:
Ok did that now.